src/Instrumentation/Laravel/composer.json

```diff
@@ -7,21 +7,21 @@
   "readme": "./README.md",
   "license": "Apache-2.0",
   "minimum-stability": "dev",
+  "prefer-stable": true,
   "require": {
     "php": "^8.0",
     "ext-json": "*",
     "ext-opentelemetry": "*",
-    "laravel/framework": ">=6.0",
+    "laravel/framework": "^6.0 || ^7.0 || ^8.0 || ^9.0 || ^10.0 || ^11.0",
```

Comment:
![image](https://github.com/open-telemetry/opentelemetry-php-contrib/assets/106102472/006fe7fb-d75f-465f-b322-3fd5cb97f467)

Source: https://laravel.com/docs/11.x/releases#support-policy

Listing the currently supported versions explicitly. I'd like to drop versions 6-9 after we make a 1.0 release of the package so that we can align with the API on PHP ^8.1.